challenges/victor/frontEndChallenge/public/app.js

```diff
@@ -0,0 +1,12 @@
+function logFormInfo(event) {
+  event.preventDefault();
+  const name = document.getElementById("name").value;
+  const email = document.getElementById("email").value;
+  const message = document.getElementById("message").value;
+  console.log(sessionStorage.getItem("captchaResponse"));
+  if (sessionStorage.getItem("captchaResponse") === "true") {
```

Comment:
How did you decide to make captchaResponse a String rather than a boolean? Are there any advantages or disadvantages to this strategy?

Reply:
I converted the captchaResponse to a string so that I could store it in sessionStorage to access later. This let me make only one call to the captcha API to verify the user early on then store that for the remainder of the session. I'm not too sure what disadvantages there could be with that method.